core/css/mobile.scss

```
@@ -32,6 +32,10 @@
 		margin-left: 0;
 	}
 
+	.skip-navigation.skip-content {
+		left: 3px;
+		margin-left: 0px;
```

```suggestion
		margin-left: 0;
```

0 does not need a unit. :) Just a detail though, great work!

---

ah, yeah, obviously... let's change that...